service_catalog/views/color.py

```diff
@@ -0,0 +1,50 @@
+from service_catalog.models.instance import InstanceState
+from service_catalog.models.request import RequestState
+
+map_dict_request_state = {
+    RequestState.ACCEPTED: "primary",
+    RequestState.NEED_INFO: "warning",
+    RequestState.SUBMITTED: "info",
+    RequestState.REJECTED: "dark",
+    RequestState.PROCESSING: "orange",
+    RequestState.COMPLETE: "success",
+    RequestState.FAILED: "danger",
+    RequestState.CANCELED: "light"
+}
+
+map_dict_instance_state = {
+    InstanceState.PENDING: "secondary",
+    InstanceState.PROVISIONING: "primary",
+    InstanceState.PROVISION_FAILED: "warning",
+    InstanceState.AVAILABLE: "success",
+    InstanceState.DELETE_FAILED: "warning",
+    InstanceState.DELETING: "primary",
+    InstanceState.UPDATING: "primary",
+    InstanceState.UPDATE_FAILED: "warning",
+    InstanceState.DELETED: "danger",
+    InstanceState.ARCHIVED: "dark",
+}
+
+map_class_to_color = {
+    "blue": "#007bff",
+    "indigo": "#6610f2",
+    "purple": "#6f42c1",
+    "pink": "#e83e8c",
+    "red": "#dc3545",
+    "orange": "#fd7e14",
+    "yellow": "#ffc107",
+    "green": "#28a745",
+    "teal": "#20c997",
+    "cyan": "#17a2b8",
+    "white": "#fff",
```

Comment:
remove white so we always see charts

Reply:
edit: do not remove it from here because it an be useful later but update your method to not use it in charts

Reply:
done